### PR TITLE
[JAVA] Codestyle check

### PR DESCRIPTION
**Pull request by likholat**

### CODE REVIEW

**Comment by likholat** [comment]
@dkurt

File: .github/workflows/code_style.yml [hunk 1]
```diff
@@ -38,3 +38,30 @@ jobs:
...
+          
+  Java:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11.0.8' # The JDK version to make available on the path.
+          java-package: jdk # (jre, jdk, or jdk+fx) - defaults to jdk
+          architecture: x64 # (x64 or x86) - defaults to x64
```
Comment (dkurt) [inline .github/workflows/code_style.yml:50]: skip if x64 is default

File: .github/workflows/code_style.yml [hunk 2]
```diff
@@ -38,3 +38,30 @@ jobs:
...
           path: code_style_diff.patch
+          
+  Java:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11.0.8' # The JDK version to make available on the path.
+          java-package: jdk # (jre, jdk, or jdk+fx) - defaults to jdk
```
Comment (dkurt) [inline .github/workflows/code_style.yml:49]: skip if default is jdk

File: .github/workflows/code_style.yml [hunk 3]
```diff
@@ -38,3 +38,30 @@ jobs:
...
+
+      - name: Check code style
+        run: |
+          wget -nc https://github.com/google/google-java-format/releases/download/google-java-format-1.9/google-java-format-1.9-all-deps.jar
+          java -jar google-java-format-1.9-all-deps.jar --set-exit-if-changed -i $(find . -type f -name "*.java" | grep ".*/inference-engine/ie_bridges/java/.*java")
+
+      - name: Create code style diff
+        if: failure()
+        run: |
+          java -jar google-java-format-1.9-all-deps.jar -r $(find . -type f -name "*.java" | grep ".*/inference-engine/ie_bridges/java/.*java") -i $(find . -type f -name "*.java" | grep ".*/inference-engine/ie_bridges/java/.*java")
```
Comment (dkurt) [inline .github/workflows/code_style.yml:60]: Why we need to runs?

File: .github/workflows/code_style.yml [hunk 4]
```diff
@@ -38,3 +38,30 @@ jobs:
...
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11.0.8' # The JDK version to make available on the path.
+          java-package: jdk # (jre, jdk, or jdk+fx) - defaults to jdk
+          architecture: x64 # (x64 or x86) - defaults to x64
+
+      - name: Check code style
+        run: |
+          wget -nc https://github.com/google/google-java-format/releases/download/google-java-format-1.9/google-java-format-1.9-all-deps.jar
+          java -jar google-java-format-1.9-all-deps.jar --set-exit-if-changed -i $(find . -type f -name "*.java" | grep ".*/inference-engine/ie_bridges/java/.*java")
```
Comment (dkurt) [inline .github/workflows/code_style.yml:55]: Is that possible to set offset to 4? Almost all the changes replace 4 whitespaces to 2.

Comment (dkurt) [inline .github/workflows/code_style.yml:55]: Why do we need `grep`?

File: .github/workflows/code_style.yml (hunk 3 above)
Comment (likholat) [reply .github/workflows/code_style.yml:60]: To change files code style and make diff

File: .github/workflows/code_style.yml [hunk 5]
```diff
@@ -38,3 +38,35 @@ jobs:
...
+          
+  Java:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11'
+
+      - uses: axel-op/googlejavaformat-action@v3
```
Comment (dkurt) [inline .github/workflows/code_style.yml:50]: Please  use wget - we cannot rely on thirdparty apps

File: .github/workflows/code_style.yml [hunk 6]
```diff
@@ -38,3 +38,28 @@ jobs:
...
+
+      - name: Check code style
+        run: |
+          wget -nc https://github.com/google/google-java-format/releases/download/google-java-format-1.9/google-java-format-1.9-all-deps.jar
+          java -jar google-java-format-1.9-all-deps.jar --set-exit-if-changed -a -i $(find . -type f -name "*.java")
+
+      - name: Create code style diff
+        if: failure()
+        run: |
+          java -jar google-java-format-1.9-all-deps.jar -a -r $(find . -type f -name "*.java") -i $(find . -type f -name "*.java")
```
Comment (dkurt) [inline .github/workflows/code_style.yml:58]: Why we have two `$(find . -type f -name "*.java")`?

File: inference-engine/ie_bridges/java/samples/benchmark_app/Main.java [hunk 7]
```diff
@@ -35,28 +33,42 @@ static boolean adjustShapesBatch(Map<String, int[]> shapes, int batchSize, Map<S
...
+                            + device
+                            + " device. Although the automatic selection usually provides a"
+                            + " reasonable performance,but it still may be non-optimal for some"
+                            + " cases, for more information look at README.");
             device_config.put(key, device + "_THROUGHPUT_AUTO");
         }
         return device_config.get(key);
-    };
+    }
+    ;
```
Comment (dkurt) [inline inference-engine/ie_bridges/java/samples/benchmark_app/Main.java:56]: What?! 😄

File: inference-engine/ie_bridges/java/samples/benchmark_app/Main.java [hunk 8]
```diff
@@ -147,7 +166,8 @@ public static void main(String[] args) {
             System.exit(1);
         }
 
-        // ----------------- 1. Parsing and validating input arguments ---------------------------------------------
+        // ----------------- 1. Parsing and validating input arguments
+        // ---------------------------------------------
```
Comment (dkurt) [inline inference-engine/ie_bridges/java/samples/benchmark_app/Main.java:170]: Please try to make it 80 symbols width

File: inference-engine/ie_bridges/java/samples/benchmark_app/Main.java [hunk 9]
```diff
@@ -352,18 +389,20 @@ public static void main(String[] args) {
...
+                || (isAsync && iteration % nireq != 0)) {
             inferRequest = inferRequestsQueue.getIdleRequest();
-            
+
             if (isAsync) {
-                // As the inference request is currently idle, the wait() adds no additional overhead 
-                //(and should return immediately).
+                // As the inference request is currently idle, the wait() adds no additional
+                // overhead
+                // (and should return immediately).
```
Comment (dkurt) [inline inference-engine/ie_bridges/java/samples/benchmark_app/Main.java:401]: Please refactor in a different way:
```java
                // As the inference request is currently idle, the wait() adds no additional
                // overhead (and should return immediately).
```

File: inference-engine/ie_bridges/java/samples/face_detection_java_sample/Main.java [hunk 10]
```diff
@@ -89,27 +89,40 @@ public static void main(String[] args) {
...
+                            + confidence
+                            + "    ("
+                            + xmin
+                            + ","
+                            + ymin
+                            + ")-("
+                            + xmax
+                            + ","
+                            + ymax
+                            + ")");
```
Comment (dkurt) [inline inference-engine/ie_bridges/java/samples/face_detection_java_sample/Main.java:120]: Please check if there is more elegant way to manage it

File: inference-engine/ie_bridges/java/samples/face_detection_sample_async/Main.java [hunk 11]
```diff
@@ -99,137 +96,163 @@ public static void main(String[] args) {
...
-                    Imgproc.putText(img, "Reading fps: " + String.format("%.3f", readFps), new Point(10, 50), 0 , 0.7, new Scalar(0, 255, 0), 1);
-                    Imgproc.putText(img, "Inference fps: " + String.format("%.3f", worksFps), new Point(10, 80), 0 , 0.7, new Scalar(0, 255, 0), 1);
+                    Imgproc.putText(
+                            img,
+                            "Reading fps: " + String.format("%.3f", readFps),
+                            new Point(10, 50),
+                            0,
+                            0.7,
+                            new Scalar(0, 255, 0),
+                            1);
```
Comment (dkurt) [inline inference-engine/ie_bridges/java/samples/face_detection_sample_async/Main.java:242]: Seems like all the arguments which take more than 80 symbols we will have this row-wise layout. Please try
```java
Imgproc.putText(img, "Inference fps: " + String.format("%.3f", worksFps),
                new Point(10, 80), 0 , 0.7, new Scalar(0, 255, 0), 1);
```

Comment (likholat) [reply inference-engine/ie_bridges/java/samples/face_detection_sample_async/Main.java]: doesn't work

File: inference-engine/ie_bridges/java/samples/benchmark_app/Main.java (hunk 7 above)
Comment (likholat) [reply inference-engine/ie_bridges/java/samples/benchmark_app/Main.java]: don't know why, but it`s error for ```};```

**Pull request closed without merging**